flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/failover/flip1/RestartPipelinedRegionFailoverStrategyBuildingTest.java

```diff
@@ -55,9 +56,9 @@ public void testIndividualVertices() {
 
 		RestartPipelinedRegionFailoverStrategy strategy = new RestartPipelinedRegionFailoverStrategy(topology);
 
-		FailoverRegion r1 = strategy.getFailoverRegion(v1.getId());
-		FailoverRegion r2 = strategy.getFailoverRegion(v2.getId());
-		FailoverRegion r3 = strategy.getFailoverRegion(v3.getId());
+		SchedulingPipelinedRegion r1 = strategy.getFailoverRegion(v1.getId());
```

Comment:
Tests in this file should be migrated to something like `PipelinedRegionComputeUtilTest`